buildSrc/src/main/kotlin/Project.kt

```diff
@@ -111,7 +111,7 @@ object Versions {
   const val mapboxGlNative = "10.7.0-beta.1"
   const val mapboxCommon = "22.1.0-beta.1"
   const val mapboxAndroidCore = "5.0.2"
-  const val mapboxAndroidTelemetry = "8.1.3"
```

Comment:
is there a matcing mapboxAndroidCore released as well?

Reply:
double checked, 8.1.4 and 5.0.2 is in the same release tag, we are good here.

https://github.com/mapbox/mapbox-events-android/tree/telem-8.1.4-core-5.0.2